components/pluralrules/src/bin/generate_res.rs

```diff
@@ -0,0 +1,19 @@
+use icu_pluralrules::data::io::json::get_resource;
```

Comment:
This is an example of a file that I hope to make obsolete with DataExporter that I'm working on in #182. If you insist on keeping bincode in this PR, can you at least a TODO to this file reminding us to delete it once #182 is landed and wired up?

Reply:
I think having `bincode` here, which matches `bincode` perf of the standalone https://github.com/zbraniecki/pluralrules will make it easier for us to measure performance of bincode `DataProvider` when we swap.

Reply:
Ok that's fine; my ask is to add a TODO since this file is in a different directory from the rest of the data scripts so it might be easy to miss.